source/Databricks/source/Jobs.UnitTests/Extensions/DependencyInjection/JobsExtensionsTests.cs

```diff
@@ -62,4 +63,20 @@ public void AddDatabricksJobs_Should_ResolveSqlClient()
         var client = serviceProvider.GetService<IJobsApiClient>();
         client.Should().NotBeNull();
     }
+
+    [Fact]
+    public void AddDatabricksJobs_Should_Resolve_Named_Http_Client()
+    {
+        // Arrange
+        var services = new ServiceCollection();
+
+        // Act
+        services.AddDatabricksJobs(_configuration);
+        var serviceProvider = services.BuildServiceProvider();
+
+        // Assert
+        var httpClientFactory = serviceProvider.GetService<IHttpClientFactory>();
+        var httpClient = httpClientFactory!.CreateClient(HttpClientNameConstants.DatabricksJobsApi);
```

Comment:
Suggestion to avoid use of `!`
```suggestion
        var httpClientFactory = serviceProvider.GetRequiredService<IHttpClientFactory>();
        var httpClient = httpClientFactory.CreateClient(HttpClientNameConstants.DatabricksJobsApi);
```

Reply:
Done.